src/math/Box3.js

```diff
@@ -468,6 +468,12 @@ Object.assign( Box3.prototype, {
 
 	translate: function ( offset ) {
 
+		if ( ! offset.isVector3 ) {
+
+			throw new Error( 'THREE.Box3: .translate() expects a Vector3.' );
```

Comment:
Same here 😇

Reply:
Do you mind removing this check? Please, don't modify files in `src/` for test PRs.